Project-I-Social-Card/social-card/src/components/CardComponents/CardBanner.js

```diff
@@ -1,2 +1,12 @@
 import React from 'react';
 import './Card.css';
+
+function CardBanner() {
+    return (
+        <div className="cardBanner">
+        <img className="reactPicture" src="https://tk-assets.lambdaschool.com/fcd75197-7d12-46ec-bc9e-4130f34822fa_reactbackground.png" alt="It's React!"></img>
```

Comment:
you should try passing this src as props so that you can reuse this component in case their is another location you would eventually like to reuse this in.